satpy/readers/eps_l1b.py

```diff
@@ -186,18 +187,21 @@ def keys(self):
             keys += val.dtype.fields.keys()
         return keys
 
-    @delayed(nout=2, pure=True)
     def _get_full_lonlats(self, lons, lats):
         nav_sample_rate = self["NAV_SAMPLE_RATE"]
         if nav_sample_rate == 20 and self.pixels == 2048:
-            from geotiepoints import metop20kmto1km
-            return metop20kmto1km(lons, lats)
+            return self._interpolate_20km_to_1km(lons, lats)
```

Comment:
Doesn't this need `da.from_delayed()`?

Reply:
it is in the calling function, but maybe it should be closer indeed.

Reply:
Fixed

Reply:
Ah, didn't check the full file, but yeah, better to have it closer for clarity.